diff --git a/node_modules/@glimmer/env/dist/commonjs/es2017/index.js b/node_modules/@glimmer/env/dist/commonjs/es2017/index.js
index f6c8ead..f8bbd3b 100644
--- a/node_modules/@glimmer/env/dist/commonjs/es2017/index.js
+++ b/node_modules/@glimmer/env/dist/commonjs/es2017/index.js
@@ -3,6 +3,6 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
-const DEBUG = exports.DEBUG = false;
+const DEBUG = exports.DEBUG = true;
 const CI = exports.CI = false;
 //# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguanMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJzcmMvaW5kZXgudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7QUFBTyxNQUFNLEFBQUssd0JBQUcsQUFBSyxBQUFDLEFBQzNCLEFBQU07QUFBQyxNQUFNLEFBQUUsa0JBQUcsQUFBSyxBQUFDIiwic291cmNlc0NvbnRlbnQiOlsiZXhwb3J0IGNvbnN0IERFQlVHID0gZmFsc2U7XG5leHBvcnQgY29uc3QgQ0kgPSBmYWxzZTtcbiJdfQ==
\ No newline at end of file
diff --git a/node_modules/@glimmer/env/dist/modules/es2017/index.js b/node_modules/@glimmer/env/dist/modules/es2017/index.js
index 88f7399..ac54674 100644
--- a/node_modules/@glimmer/env/dist/modules/es2017/index.js
+++ b/node_modules/@glimmer/env/dist/modules/es2017/index.js
@@ -1,3 +1,3 @@
-export const DEBUG = false;
+export const DEBUG = true;
 export const CI = false;
 //# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiaW5kZXguanMiLCJzb3VyY2VSb290IjoiIiwic291cmNlcyI6WyJzcmMvaW5kZXgudHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6IkFBQUEsTUFBTSxDQUFDLE1BQU0sS0FBSyxHQUFHLEtBQUssQ0FBQztBQUMzQixNQUFNLENBQUMsTUFBTSxFQUFFLEdBQUcsS0FBSyxDQUFDIiwic291cmNlc0NvbnRlbnQiOlsiZXhwb3J0IGNvbnN0IERFQlVHID0gZmFsc2U7XG5leHBvcnQgY29uc3QgQ0kgPSBmYWxzZTtcbiJdfQ==
\ No newline at end of file
