diff --git a/node_modules/els-addon-glint/index.js b/node_modules/els-addon-glint/index.js
index e7112d4..cd45632 100644
--- a/node_modules/els-addon-glint/index.js
+++ b/node_modules/els-addon-glint/index.js
@@ -41,6 +41,24 @@ function buildHelpers({ languageServer , documents , connection  }) {
                     if (message.includes('No overload matches this call.')) {
                         return false;
                     }
+                    if (message.includes("Object is of type 'unknown'")) {
+                        return false;
+                    }
+                    if (message.includes("type 'unique symbol'")) {
+                        return false;
+                    }
+                    if (message.includes("does not exist on type 'EmptyObject'")) {
+                        return false;
+                    }
+                    if (message.includes("is not assignable to parameter of type 'EmptyObject'.")) {
+                        return false;
+                    }
+                    if (message.includes("is not assignable to parameter of type 'Element'")) {
+                        return false;
+                    }
+                    if (message.includes("Type 'unknown' is not assignable to type 'SafeString'")) {
+                        return false;
+                    }
                     return true;
                 });
                 if (filteredDiagnostics.length) {
@@ -143,6 +161,9 @@ module.exports = class ElsAddonQunitTestRunner {
     }
     async onComplete(_, params) {
         const results = await this.languageServer.getCompletions(params.textDocument.uri, params.position);
+        if (results.length > 50) {
+            return params.results;
+        }
         return [
             ...results || [],
             ...params.results
